config/dev.exs

```diff
@@ -59,6 +59,7 @@ config :uppy, UppyWeb.Endpoint,
 
 # Watch static and templates for browser reloading.
 config :uppy, UppyWeb.Endpoint,
+  server: true,
```

Comment:
I would leave this out because then you can run iex without the server starting which is sometimes useful.
And you can always do `iex -S mix phx.server` if you need it.